components-registry-automation/src/main/kotlin/org/octopusden/octopus/components/registry/automation/ComponentsRegistryCommand.kt

```
@@ -0,0 +1,44 @@
+    private val context by findOrSetObject { mutableMapOf<String, Any>() }
+
+    override fun run() {
+        logger.info("Components Registry Service URL: $url")
+        try {
+            val clientUrlProvider = object : ClassicComponentsRegistryServiceClientUrlProvider {
+                override fun getApiUrl(): String = url
+            }
+            val client = ClassicComponentsRegistryServiceClient(clientUrlProvider)
+            context[LOGGER] = logger
+            context[CLIENT] = client
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)